modules/ROOT/pages/_partials/push/unregistering_device.adoc

```diff
@@ -0,0 +1,83 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+[id='unregistering-device']
+= Unregistering device on {unifiedpush-service} Service
+
+This section describes how to unregister devices.
+
+[tabs]
+====
+// tag::excludeDownstream[]
+Android::
++
+--
+. Getting an instance of the PushService
++
+[source,java]
+----
+PushService pushService = new PushService.Builder().openshift().build();
+----
+
+. Registration request
++
+[source,java]
+----
+pushService.unregisterDevice()
+  .requestOn(new AppExecutors().mainThread())
+  .respondWith(new Responder<Boolean>() {
+    @Override
+    public void onResult(Boolean value) { // Yay }
+
+    @Override
+    public void onException(Exception exception) { // Oops! }
+});
+----
+
+--
+iOS::
++
+--
+
+This feature is not yet supported in the AeroGear iOS SDK
```

Comment:
Maybe we should rephrase it. This phrase makes me think that we did not implement it yet, but actually it's not supported by iOS/Apple